File: 태성/0623/행렬의덧셈.py
def solution(arr1, arr2):
    answer = []

    for i in range(len(arr1)):
        pre = []
        for j in range(len(arr1[i])):
            pre.append(arr1[i][j] + arr2[i][j])

        answer.append(pre)
    return answer

File: 태성/LV2/0623/행렬의덧셈.js
function solution(arr1, arr2) {
    var answer = [];
    
    for (let i_1 in arr1){
        var pre = []
        for (let i_2 in arr1[i_1]){
            pre.push(arr1[i_1][i_2]+arr2[i_1][i_2])
            
        }
        answer.push(pre)
    }
    return answer;
}